2007-08-01  Jakub Jelinek  <jakub@redhat.com>

	* sysdeps/unix/sysv/linux/kaio_misc.h (AIO_MISC_NOTIFY,
	AIO_MISC_WAIT): Add LLL_PRIVATE as last argument to lll_futex_*wait.

--- libc/rtkaio/sysdeps/unix/sysv/linux/kaio_misc.h	2006-09-07 12:36:11.000000000 +0200
+++ libc/rtkaio/sysdeps/unix/sysv/linux/kaio_misc.h	2007-08-01 12:21:31.000000000 +0200
@@ -55,7 +55,7 @@
 # define AIO_MISC_NOTIFY(waitlist) \
   do {									      \
     if (*waitlist->counterp > 0 && --*waitlist->counterp == 0)		      \
-      lll_futex_wake (waitlist->counterp, 1);				      \
+      lll_futex_wake (waitlist->counterp, 1, LLL_PRIVATE);		      \
   } while (0)
 
 # define AIO_MISC_WAIT(result, futex, timeout, cancel)			      \
@@ -74,7 +74,8 @@
 	int status;							      \
 	do								      \
 	  {								      \
-	    status = lll_futex_timed_wait (futexaddr, oldval, timeout);	      \
+	    status = lll_futex_timed_wait (futexaddr, oldval, timeout,	      \
+					   LLL_PRIVATE);		      \
 	    if (status != -EWOULDBLOCK)					      \
 	      break;							      \
 									      \
